### ISSUE TITLE
Fossify Phone (Contact Tab) and Contacts App not showing any contacts

### ISSUE THREAD

**Issue by Kasi07** [FossifyOrg/Phone#28]
**Checklist**
 - [x] I can reproduce the bug with the latest version. 
 - [x] I made sure that there are no existing issues - open or closed - to which I could contribute my information to.
 - [x]  I have read the FAQ and my problem isn't listed.
 - [x]  I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
 - [x]  This issue contains only one bug.
 - [x]  I have read and understood the [contribution guidelines.](https://github.com/FossifyOrg/General-Discussion?tab=readme-ov-file#faq)

**Describtion**
No Contacts are visible. The issue started with one of the later versions of the former Simple-Dialer and Simple-Contacts app and seems to persist here. Strangely The correct names are shown in the call history. There was an issue for Simple-Contacts (SimpleMobileTools/Simple-Contacts#1044) which seems to be the same issue.
As mentioned in a comment [here](https://github.com/SimpleMobileTools/Simple-Contacts/issues/1044#issuecomment-1759556701) the versions Simple-Dialer 5.18.0 and Simple-Contacts 6.22.4 are working as intended. I did double check and this holds true for me too.

I suspect this will be a problem in the Fossify Contacts app too.

**To Reproduce**
1. Open Phone
2. see no contacts

**Expected behavior**
See all contacts.

**Screenshots**
Screenshots can be found in the old issue here: SimpleMobileTools/Simple-Contacts#1044

**Helpful Links**
- https://github.com/SimpleMobileTools/Simple-Contacts/compare/6.22.4...6.22.5
- https://github.com/SimpleMobileTools/Simple-Dialer/compare/5.18.0...5.18.1

**Device info (please complete the following information):**
 - Model: Pixel 7
 - OS: GrapheneOS Android 14
 - App version 1.0.0

**Comment by Aga-C** [comment]
In which memory do you keep contacts?

**Comment by Kasi07** [comment]
I am using https://github.com/bitfireAT/davx5-ose to access them via carddav.

**Comment by Aga-C** [comment]
Thanks! Please just change the title that you're writing about Contacts tab. Phone doesn't use Contacts app to get the contacts, these are two separate apps.

**Comment by Kasi07** [comment]
As the Fossify Contact app has just been released, I have been able to confirm that this bug is also present in the Contact app.

**Comment by ronniedroid** [comment]
Just want to add that I did not have this issue on android 13, only on android 14 (custom rom)

**Comment by Rewertius** [comment]
Confirm, can't see my synced Contacts. App Version 1.0.1.
Sync with davx5 from mailbox.org via carddav on Pixel 6a with GrapheneOS (Android 14)
With the native contacts app, there is no problem.

**Comment by rightsaidfred99** [comment]
It also affects when receiving a call, no saved contact displays, only their real number.

**Comment by nucular8** [comment]
I can confirm that after upgrading from LOS 20 (Android 13) to LOS 20 (Android 14) my synced contacts are not shown any more. 
The filter only offers to choose from local storage and device storage. 
Contacts app is affected in the same way, I just opened an issue. 
PS. Sync is done with DAVX5.

**Comment by czarnyckm** [comment]
I have the same problem with CARDDAV contacts.
Works fine with 'koler' or 'contacts you'
as well as 'Simple Contacts Pro SE' f-droid version 6.22.4.2 Pro (the Github version is broken since January update)

**Comment by dorianve** [comment]
I'm experiencing a similar issue. For me some synced contacts accounts are showing (with Davx), but I have contacts from tutanota (tuta) and those are not showing whereas they appear in the Google Contacts app (which I'd rather not use).

Fossify Contacts 1.0.1

**Comment by oliva** [comment]
Just a note: this also affects the calendar app's contact birthday/anniversary import feature

**Comment by bb1a2** [comment]
Could this be (highly) prioritized, please? Incoming calls are shown as numbers only and I can't tell who is calling.

**Comment by lmcj-xyz** [comment]
My contacts are synced using davx5, and this is a problem for Fossify Contacts and Fossify Dialer but only after activating a new profile.

**I have two profiles: the main profile I don't run any Google services nor apps from the Play Store, that is the profile with the problem. In the profile where I run Google services I installed the Fossify Contacts app and it displays my contacts just fine.**

Before using two profiles I was running the sandboxed google play services, and it was all fine.

As others report, GrapheneOS' contacts and dialer apps can see my contacts form davx5 just fine.

**Comment by deepakdhanji** [comment]
I'd like to add that, the Phone app is unable to read contacts from AOSP Contacts which has DavX based storage. 

Not sure if in order to use Fossify Phone App, I have to use Fossify Contacts as well or can use any other Contacts App.

**Comment by lmcj-xyz** [comment]
In addition to my [comment above](https://github.com/FossifyOrg/Phone/issues/28#issuecomment-2065257687), it's worth to notice that only Phone and Contacts fail to see my davx5 contacts after uninstalling the sandboxed Google Play services, but Messages can see them without a problem.

**Comment by bb1a2** [comment]
I figured out why it is happening in my phone. Storage location. Contacts could see it but Phone couldn't (it was missing from the list).

**Comment by naveensingh** [comment]
There is a small chance that this might be fixed in version 1.2.0: https://github.com/FossifyOrg/Phone/releases/tag/1.2.0

**Comment by Kasi07** [comment]
Same for me.

**Comment by naveensingh** [comment]
I could not reproduce the issue on two Android 14 devices. I can't install GrapheneOS at the moment but I have a few ideas so I'll need someone to test things for me. Here is another APK with a potential fix:

[phone-4-foss-release-issue-28.zip](https://github.com/FossifyOrg/Phone/files/15244901/phone-4-foss-release-issue-28.zip)

(GitHub doesn't support APKs so just rename the .zip to .apk and install, do not extract it)

**Comment by Kasi07** [comment]
Still does not work for me.

**Comment by dorianve** [comment]
@irgendwie , actually, I think I should mention that I'm on Android 11, stock version (never tinkered with... so far...), with Google Play services enabled, and I'm affected as well ... Interesting.

**Comment by chesio** [comment]
> @irgendwie , actually, I think I should mention that I'm on Android 11, stock version (never tinkered with... so far...), with Google Play services enabled, and I'm affected as well ... Interesting.

Your problem however seems to be related to syncing contacts from specifing app (Tutanota) or am I mistaken? Cause you've written above:

> I'm experiencing a similar issue. For me some synced contacts accounts are showing (with Davx), but I have contacts from tutanota (tuta) and those are not showing whereas they appear in the Google Contacts app (which I'd rather not use).

**Comment by dorianve** [comment]
@chesio indeed. I have 2 contact accounts syncing on this phone:
- a "dav" account syncing with DAVx5. This one is working fine
- another contact account from Tuta (not showing in the fossify apps, but showing on the Google contact app)

My understanding was that Fossify was syncing OK in most cases (eg. My "DAVx5 case"), and other times not (eg. My "Tuta case" **and other people's issues in this thread**).

Do you think it's actually 2 different issues?

**Comment by chesio** [comment]
> My understanding was that Fossify was syncing OK in most cases (eg. My "DAVx5 case"), and other times not (eg. My "Tuta case" **and other people's issues in this thread**).
> 
> Do you think it's actually 2 different issues?

I would say it is a different issue although it manifest itself in the same way (no contact showing).

Almost all of the commenters in this thread are reporting having problems on Android 14 with custom ROM. Your problem sounds different, so I would recommend that you open a new issue. It might be easier to solve and if someone else is also having issues with Tutanota app, a dedicated issue will be easier to find for them.

**Comment by sneakybear** [comment]
Same issue here with latest Graphene on a 6a.
Profile with no Google services = no contacts
Profile with sandboxed Google = contacts
I am able to create contacts fine in both.

**Comment by samuelion** [comment]
Same problem here. No problem with stock app. 
More funny, no problem with fossify SMS (i know it is anorher app with not so much link with this one, but all my contact are suggested when I start writing a new sms, thus no difficulties to get them while it is not the case with fossify phone and agenda !)

**Comment by chept** [comment]
Same here.

Google Pixel 7 Pro wtih GrapheneOS

contacts synchronized with a davx account

No problem with stock app

Fossify Contacts show contacts only when Google Play Services installed

**Comment by PalmTreeVI** [comment]
Added comment in Fossify Contacts which is similar issue and likely related. [Comment](https://github.com/FossifyOrg/Contacts/issues/83#issuecomment-2415557030)

**Comment by pern0x** [comment]
Same issue here on grapheneos with android 15. After removing all google services, it shows only contacts from simcard in contacts tab. History shows all calls and names of contacts, but on contact tab they are not shown.

**Comment by naveensingh** [comment]
It should be fixed in the next update.

**Comment by pern0x** [comment]
@naveensingh: Thanks. I tried it with android studio. After the last update the contacts are shown.

**Comment by naveensingh** [comment]
It's out now: https://github.com/FossifyOrg/Phone/releases/tag/1.3.0
